tests/osaka/eip7594_peerdas/helper_functions.py

```diff
@@ -0,0 +1,240 @@
+"""
+abstract: Tests [EIP-7594: PeerDAS - Peer Data Availability Sampling](https://eips.ethereum.org/EIPS/eip-7594)
+    Tests [EIP-7594: PeerDAS - Peer Data Availability Sampling](https://eips.ethereum.org/EIPS/eip-7594).
+"""  # noqa: E501
+import base64 as b64
+import json
+import math
+
+import ckzg
+
+from spec import Spec, ref_spec_7594
+
+REFERENCE_SPEC_GIT_PATH = ref_spec_7594.git_path
+REFERENCE_SPEC_VERSION = ref_spec_7594.version
+
+def bytes_from_hex(hex_string):
+    """Convert a hex string to bytes."""
+    return bytes.fromhex(hex_string.replace("0x", ""))
+
+def generate_blob_from_hex_byte(hex_byte: str) -> bytes | None:
+    """Take a singular hex byte string and return a valid blob built from this byte."""
+    # preparation (remove 0x if present)
+    if "0x" in hex_byte:
+        hex_byte = hex_byte.replace("0x", "", 1)
+
+    # validity checks
+    if len(hex_byte) != 2:
+        print("You should have passed a singular hex byte like e.g. fe to this function")
+        return None
+
+    if int(hex_byte, 16) > 115:
+        # TODO: figure out why this happens
+        print("For some reason bytes larger than this will lead to errors later in compute_cells(), so it's not allowed for now.")  # noqa: E501
+        return None
+
+    hex_byte = hex_byte.lower()
+
+    valid_chars = "0123456789abcdef"
+    if (hex_byte[0] not in valid_chars) or (hex_byte[1] not in valid_chars):
+        print("You should have passed a singular hex byte consisting of only two chars in 0-9 and a-f like e.g. fe to this function")  # noqa: E501
+        return None
+
+
+    # create blob
+    exp = int(math.log2(Spec.BYTES_PER_BLOB))
+    blob_string: str = "0x" + hex_byte * (2**exp)
+    blob: bytes = bytes_from_hex(blob_string)
+
+    return blob
+
+
+def eest_blob_to_kzg_commitment(blob: bytes) -> bytes:
+    """Take a blob and returns a cryptographic commitment to it. Note: Each cell seems to hold a copy of this commitment."""  # noqa: E501
+    # sanity check
+    expected_blob_length = 2**int(math.log2(Spec.BYTES_PER_BLOB))
+    assert len(blob) == expected_blob_length, f"Expected blob of length {expected_blob_length} but got blob of length {len(blob)}"  # noqa: E501
+
+    # calculate commitment
+    ts = ckzg.load_trusted_setup("trusted_setup.txt", 0)
+    commitment = ckzg.blob_to_kzg_commitment(blob, ts)
+
+    assert len(commitment) == Spec.KZG_COMMITMENT_LENGTH, f"Expected {Spec.KZG_COMMITMENT_LENGTH} resulting commitments but got {len(commitment)} commitments"  # noqa: E501
+
+    return commitment
+
+
+def eest_compute_cells(blob: bytes) -> list[int]:
+    """Take a blob and returns a list of cells that are derived from this blob."""
+    ts = ckzg.load_trusted_setup("trusted_setup.txt", 0)
+
+    cells = ckzg.compute_cells(blob, ts)
+
+    assert len(cells) == 128
+
+    return cells
```

Comment:
Several of these wrapper functions don't appear to be very useful. Could we not just call ckzg directly?